configs/example_configs_validation.py

```diff
@@ -0,0 +1,50 @@
+import os
```

Comment:
im wondering if this would be better in `/tools/config_validation`

Reply:
its quite handy having it next to the configen.sh script which pulls in similar files, so ill leave it here for now

i guess my reasoning for putting in tools is that it effects dirs outside its own root